Python/.gitattributes

```diff
@@ -0,0 +1 @@
+*.ipynb linguist-vendored
```

Comment:
Can we have a global `gitattributes` ?

Reply:
The reason I did not do that was because this specific attribute is applicable to Python code only\; thus keeping in line with segregating specific stuff to specific directory levels. Is the repo-wide file preferred?